lib/pharos/configuration/host.rb

```diff
@@ -82,6 +82,9 @@ def labels
         labels = @attributes[:labels] || {}
 
         labels['node-address.kontena.io/external-ip'] = address
+        pa = private_address || private_interface_address
+        labels['node-address.kontena.io/internal-ip'] = pa if pa
```

Comment:
this could be also done as:
```
        labels['node-address.kontena.io/external-ip'] = address
        labels['node-address.kontena.io/internal-ip'] = private_address || private_interface_address

        labels['node-role.kubernetes.io/worker'] = '' if worker? && !labels.find{ |k, _| k.to_s.start_with?("node-role.kubernetes.io") }

        labels.compact
```

dunno if it would be any cleaner that way?